src/libs/helpers/state-manipulator/asset-manipulator.ts

```diff
@@ -0,0 +1,57 @@
+import { bool } from "@polkadot/types-codec";
+import { bnToHex, hexToBigInt, nToHex } from "@polkadot/util";
+import Debug from "debug";
+import { ALITH_ADDRESS, USDT_ASSET_ID } from "../../../utils/constants";
+import {
+  Action,
+  encodeStorageBlake128MapKey,
+  encodeStorageBlake128DoubleMapKey,
+  encodeStorageKey,
+  StateManipulator,
+  StateLine,
+} from "./genesis-parser";
+const debug = Debug("helper:balances-manipulator");
+
+export class AssetManipulator implements StateManipulator {
+  account: string;
+  assetId: string;
+  amount: bigint;
+  injected: boolean;
+  key: string;
+  private readonly assetsGeneralPrefix = encodeStorageKey("Assets", "Account");
+
+  // TODO: Increase asset total issuance by injected amount
```

Comment:
Yes, we should always try to get the state consistent for all the modications we do

Reply:
Done